setup.py

```diff
@@ -121,6 +95,20 @@ def get_include_dirs():
 
     return include_dirs
 
+# TODO Do we need to determine package data dynamically? If so, should
```

Comment:
I agree to to leave it as it is. You are responsible for the conda pkg, right? So now you are aware and the TODO will help you remember if you or someone else creates the PR